src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1439,7 +1447,8 @@ public void spawnCitizen(final CitizenData data)
                     //TODO: add Colony Name prefix?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)